gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "12.10.21",
```

Comment:
Version bumps not expected; this is being merged to the feature branch

Reply:
> Version bumps not expected; this is being merged to the feature branch

It was only to trigger CI. Reverted here: c078212807fd8ba661742406d1282efbed3d16d5